microprot/snakemake/Snakefile

```diff
@@ -107,130 +107,115 @@ def split_x(inp_0, out_0, step="NA", version=config['VERSION'],
         shell('echo "pass" > {out_0}')
 
 
+def copy_out(source_dir='01-PDB', dest_dir='PDB', tempdir='/dev/null'):
```

Comment:
do we want to copy or can we move to same space? How about softlinks to avoid redundancy?

Reply:
this function is used to copy from localscratch to the main filesystem, hence the use of rsync to package the data on its way.
so softlinks are a no-go and `move = copy+delete`, hence it's not needed because I do a bulk clean-up later on.